webpack/assets/javascripts/jquery.ui.custom_spinners.js

```diff
@@ -105,16 +105,22 @@ $(() => {
   });
 });
 
-export function initAll() {
-  initByte();
-  initCounter();
+export function initAll(selection = null) {
+  initByte(selection);
+  initCounter(selection);
 }
 
-export function initCounter() {
-  $('input.counter_spinner').each(function() {
+export function initCounter(selection = null) {
+  // might be called with Event-argument, if initAll is used as EventHandler
+  const sel =
+    (selection !== null && selection.selector) === undefined ? null : selection;
```

Comment:
Why do we have this here? Wouldn't it be better to call the input variable `context` and pass it right to the jquery function?
If `initAll` is a event handler, than I would ragher see this in the initAll, and I would guess right way would be to ingnore it in such case, as it usually can be just a mistake. And I would prefer filter out event arguments than the only accepting jQuery objects ( jQuery context might be a string, or even Dom object ).

Reply:
I agree that filtering out EventObjects is the better approach here.
Is there a way to reliantly check that an object is an EventObject?
I must admit I always struggle in Javascript & Co. with this stuff :flushed:

Reply:
I totaly understand, it is bad design if we need to do that actually. :(
But I know there are usages like that throughout our code base :frowning_face:
Having in mind, that we would like to have React component for the spinners in near future, I believe suboptimal solution is sufficient. I came up with [1]:
```javascript
if ( typeof event  ==='object' selection.hasOwnProperty('originalEvent') ) { selection = null; }
```

[1] https://stackoverflow.com/a/27066820

Reply:
I played around a little bit and I came to the conclusion that the following check seems to identify the jQuery Events that I got:
```javascript
selection.constructor === $.Event
```

Reply:
I played around a little and found that checking the object's constructor for identity with `jquery.Event` seems to do the trick.